scenes/game/game.gd

```diff
@@ -90,6 +90,7 @@ var cached_opponent_location = 0
 var reference_popout_toggle_enabled = false
 var reference_popout_toggle = false
 var opponent_cards_before_reshuffle = []
+var boost_selection_saved = {}
```

Comment:
I feel like there has to be a better name.
Maybe boost_selection_options or something.